rails.rb

```diff
@@ -19,6 +32,22 @@
 # returns a grade for the score, either "A", "B", "C", "D", or "F".
 # assign_grade(90) should return A
 # assign_grade(75) should return C
+ 
+ def grades score=0 
+     grade="study"
+     if score >= 90 
+         grade= "A"
+    elsif score>= 80
+        grade ="B"
+    elsif score >= 70
+        grade= "C"
+    elsif score >= 60
+        grade ="D"
+    else score>= 50
+        grade ="F"
+end 
+    grade 
+puts grade (54)
 
```

Comment:
This looks good, just remember your end after the else and you still have an extra "grade" at the end there.